src/components/views/elements/MessageEditor.js

```diff
@@ -166,6 +166,17 @@ export default class MessageEditor extends React.Component {
             contentBody.format = newContent.format;
             contentBody.formatted_body = ` * ${newContent.formatted_body}`;
         }
+
+        // if nothing has changed then bail
```

Comment:
Thanks for working on this! Do you think this approach will be reliable for more complicated, formatted messages? As `newContent.formatted_body` is the result of html > md > html conversion, I can imagine it could easily introduce at least one byte of difference. Have you maybe considered something like a `dirty` flag that is set in onInput?

Reply:
So yes a dirty flag would be the simplest approach, but it would not skip if the message was edited then <kbd>Ctrl</kbd>+<kbd>Z</kbd> then <kbd>Enter</kbd>

Reply:
Atm the new editor doesn't support undo well, but once it does we could consider `editor.hasUndoSteps` to be the dirty flag ...

Reply:
turns out the class already had a dirty flag